client/features/media/MediaDetail.jsx

```diff
@@ -4,15 +4,15 @@ import './MediaDetail.css';
 
 const MediaDetail = props => (
```

Comment:
my mediaDetail component looks drastically different now in the PR I submitted. Will approving your affect mine? Or I will just have merge conflicts to deal with?

Reply:
Yours was already approved, so it's my problem now :)